suites/manage/ocp/test_pvc_invalid_inputs.py

```diff
@@ -0,0 +1,135 @@
+import logging
+import pytest
+
+from ocs import defaults
+from ocsci.config import ENV_DATA
+from ocsci.testlib import tier1, ManageTest
+from resources.ocs import OCS
+from tests import helpers
+from ocs.ocp import OCP
+from ocs.exceptions import CommandFailed
+
+log = logging.getLogger(__name__)
+
+OCCLI = OCP(kind='service', namespace=ENV_DATA['cluster_namespace'])
+
+SC_OBJ = None
+SC_NAME = None
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    This is a test fixture
+    """
+    self = request.node.cls
+
+    def finalizer():
+        teardown()
+    request.addfinalizer(finalizer)
+    setup(self)
+
+
+def setup(self):
+    """
+    Setting up the environment for the test
+    """
+    # Create a storage class
+    log.info("Creating a Storage Class")
+    self.sc_data = defaults.STORAGECLASS_DICT.copy()
+    self.sc_data['metadata']['name'] = helpers.create_unique_resource_name(
+        'test', 'csi-rbd'
+    )
+    global SC_OBJ
+    global SC_NAME
+    SC_OBJ = OCS(**self.sc_data)
+    SC_NAME = self.sc_data['metadata']['name']
```

Comment:
2 things here:
1) this is a global parameter and therefore not need to return it as it is accessible everywhere in this file
2) why to keep it if you can access it everywhere with self.sc_data['metadata']['name']?